src/human_aware_rl/rllib/rllib.py

```diff
@@ -93,15 +94,24 @@ def action(self, state):
         my_obs = obs[self.agent_index]
 
         # Use Rllib.Policy class to compute action argmax and action probabilities
-        [action_idx], rnn_state, info = self.policy.compute_actions(
+        # The first value is action_idx, which we will recompute below so the results are stochastic
+        _, rnn_state, info = self.policy.compute_actions(
             np.array([my_obs]), self.rnn_state
         )
-        agent_action = Action.INDEX_TO_ACTION[action_idx]
 
         # Softmax in numpy to convert logits to normalized probabilities
         logits = info["action_dist_inputs"]
         action_probabilities = softmax(logits)
 
+        # reseed to make sure the outputs are stochastic
+        random.seed()
```

Comment:
See convos on slack re: this line

Reply:
Yeah sorry you are right. I was looking at the wrong results when I was testing it that night and got myself confused.